File: Tugas coding modul 6/coding c/PRAK603-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void guru(int n1, int n2, int kali[n1], int kali1[n2]) { 
    for (int i = 0; i < n1; i++) {
        printf("%d ", kali[i] * kali1[i]);
    }
    printf("\n");
}

int main() {
    int n1, n2;

    scanf("%d %d", &n1, &n2);

    if (n1 != n2) {
        printf("\nJumlah tidak sama \n");
        return 0;
    }

    int kali[n1];
    for (int i = 0; i < n1; i++) {
        scanf("%d", &kali[i]);
    }

    int kali1[n2];
        for (int j = 0; j < n2; j++) {
            scanf("%d", &kali1[j]);
        }

        printf("\n");
        guru(n1, n2, kali, kali1);

    return 0;
}

File: Tugas coding modul 6/coding py/PRAK603-2310817120005-Noviana_Nur_Aisyah.py
def guru(n1, n2, kali, kali1):
    if n1 != n2:
        print("\nJumlah tidak sama \n")

    for i in range(n1):
        print(kali[i] * kali1[i], end=" ")

n1, n2 = map(int, input().split())
if n1 != n2:
    print('\nJumlah tidak sama')
    exit()

kali = [int(j) for j in input().split()]
kali1 = [int(j) for j in input().split()]

print()
guru(n1, n2, kali, kali1)